src/uwtools/drivers/fv3.py

```diff
@@ -0,0 +1,321 @@
+"""
+A driver for the FV3 model.
+"""
+
+import os
+import stat
+from datetime import datetime
+from pathlib import Path
+from shutil import copy
+from typing import Any, Dict
+
+from iotaa import asset, dryrun, external, task, tasks
+
+from uwtools.config.formats.fieldtable import FieldTableConfig
+from uwtools.config.formats.nml import NMLConfig
+from uwtools.config.formats.yaml import YAMLConfig
+from uwtools.drivers.driver import Driver
+from uwtools.logging import log
+from uwtools.types import DefinitePath
+from uwtools.utils.file import resource_pathobj
+from uwtools.utils.processing import execute
+
+
+class FV3(Driver):
+    """
+    A driver for the FV3 model.
+    """
+
+    def __init__(
+        self, config_file: DefinitePath, cycle: datetime, dry_run: bool = False, batch: bool = False
+    ):
+        """
+        The FV3 driver.
+
+        :param config_file: Path to config file.
+        :param cycle: The forecast cycle.
+        :param dry_run: Run in dry-run mode?
+        :param batch: Run component via the batch system?
+        """
+        super().__init__(config_file=config_file, dry_run=dry_run, batch=batch)
+        self._config.dereference(context={"cycle": cycle})
```

Comment:
Here's the new line that renders `cycle` Jinja2 references in the input YAML config.